macros/sql/get_tables_by_prefix_sql.sql

```diff
@@ -5,7 +5,7 @@
 {% macro default__get_tables_by_prefix_sql(schema, prefix, exclude='') %}
 
         select distinct 
-            table_schema || '.' || table_name as ref
+            table_schema as "table_schema", table_name as "table_name"
```

Comment:
Should we try to update this macro to accept a database name? As it stands, `get_tables_by_prefix` accepts a database, but this macro only queries for tables in the database that dbt is connected to

Reply:
Good call, will update